src/screens/import-account/import-wallet/import-wallet.form.tsx

```diff
@@ -13,3 +17,29 @@ export const importWalletValidationSchema: SchemaOf<ImportWalletFormValues> = ob
 export const importWalletInitialValues: ImportWalletFormValues = {
   seedPhrase: ''
 };
+
+export type ImportKukaiWalletFormValues = {
+  keystoreFile: FileInputValue;
+  password: string;
+  shouldUseFilePasswordForExtension?: boolean;
+};
+
+export const importKukaiWalletValidationSchema: SchemaOf<ImportKukaiWalletFormValues> = object().shape({
+  keystoreFile: object().test(
+    'keystore-file',
+    'A keystore file is required',
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    (value: unknown) => typeof value === 'object' && isDefined(value) && isString((value as any).uri)
```

Comment:
what documentation says about typisation?

Reply:
could you set `FileInputValue` as `value` type?

Reply:
I haven't looked at documentation
Thanks for paying attention, I can create a typeguard function to reduce the amount of kludges

Reply:
Or do something else, I don't know for now